### DISCUSSION TITLE
Typo on Configuration wiki

### DISCUSSION THREAD

**Discussion by meeki007** [vladmandic/human#74] [General]
https://github.com/vladmandic/human/wiki/Configuration

blazeface-front listed twice. 
```
modelPath: '../models/blazeface-back.json', // can be 'blazeface-front', 'blazeface-front' or 'faceboxes'
```

should be
```
modelPath: '../models/blazeface-back.json', // can be 'blazeface-front', 'blazeface-back' or 'faceboxes'
```

also added pull request for https://github.com/vladmandic/human/edit/main/config.js

pull request ---> https://github.com/vladmandic/human/pull/75

**Comment by vladmandic** [comment]
merged. btw, this should be an issue not a conversation as issues can be closed with successful merge.

**Reply by meeki007** [reply]
understood. I tired to make a push to the wiki and it sent me back to the wiki. I assumed the wiki might be under a diff document I had no access to.

Next time ill bring that up in the push .... my bad just trying to contrib :P

**Comment by vladmandic** [comment]
how github handles wiki is that it's a separate sub-repository, it's not part of the main repository itself.
(if it weren't for github handling, i would have made it just pages within main repository)
